### ISSUE TITLE
Project Update- 10-05

### ISSUE THREAD

**Issue by cdornn** [cmarkulin/hip-hop-humanities#2]
# Project Update for Meeting on Tues. 10-05:
___
Today our group met for the second time. We finalized our list of artists down to twelve names: Tyler the Creator, Kanye West, Mac Miller, Kendrick Lamar, J. Cole, Travis Scott, Vince Staples, Earl Sweatshirt, MF DOOM, A$AP Rocky, and Kid Cudi. It was then decided that of these twelve artists, five songs would be taken from their first album as well as five more songs from their latest, making 10 songs per artist. Potentially, we may use a random number generator to pick these songs so that we may avoid bias. In order to establish some organization for the many songs that we will be marking up, we decided on having individual files for each song, which we would then place in subfolders related to their respective artist.

**Comment by MasonGobat** [comment]
It is interesting that you are doing so many artists, as well as so many songs. I wonder though, if it would be helpful to include songs from an interim album so that perhaps a transformation can be seen as developing. This would add a lot more songs, so I can see why it is not an appealing idea. Is your research also going to look at the size of each artist, especially since some of those names I have never heard of and wonder if their popularity is less than others and what effect that may have?

**Comment by ShaneMoniot** [comment]
I agree a little bit with what both people before me said. I also find it interesting that you've chosen to do so many artists and songs. To build off of that, would it make sense to look at less artists but to look at all of their songs in each album? I know you said that you want to try to remove bias by using a random number generator, but I'm not sure there really is too much bias when picking songs from the albums themselves, and it might provide more context if you use all of the songs from the albums. If you want to try to eliminate bias I think a good way to do so would be to use a rng to select which artists you are going to look at.

**Comment by amm492** [comment]
This is such an exciting project, especially because the material probably hits home for most of you. I understand that want to see how the artists have grown by picking songs from their first album and their most recent, but have you guys thought about maybe finding lists, that have already been created, of their songs from least to most popular to see how/ where those fit in? I know this breaks away from your question a bit. What makes lyrics of newer songs better than others? Just some things to think about! Can't wait to see where this is headed!
